src/logic/calculate.js

```diff
@@ -0,0 +1,16 @@
+import operate from './operate';
+
+const calculate = (dataObj, buttonName) => {
+  const { total, next, operation } = dataObj;
+
+  if (buttonName === '+/-') {
+    if (total) {
+      return total * -1;
+    }
+    return next * -1;
+  }
+
+  return operate(total, next, operation);
+};
```

Comment:
### Heads-Up:
** I think you know you need to add more cases, perhaps you want to add them in the next milestone** 
let me point out these 2 👇 , students forget this a lot.

1. if the `buttonName` = A/C`, we should return `0`
2. if the `buttonName`  = `.` we should return `0.`

Reply:
Thank you for heads-up. I'll surely implement those features.